src/domains/nodo-common/06_service_bus_wisp_converter.tf

```diff
@@ -0,0 +1,70 @@
+resource "azurerm_servicebus_namespace" "wisp_converter_servicebus" {
+  count = var.enable_wisp_converter ? 1 : 0
+
+  name                = "${local.project}-servicebus"
+  location            = azurerm_resource_group.wisp_converter_rg[0].location
+  resource_group_name = azurerm_resource_group.wisp_converter_rg[0].name
+  sku                 = var.wisp_converter_service_bus.sku
+
+  tags = var.tags
+
+  depends_on = [
+    azurerm_resource_group.wisp_converter_rg
+  ]
+}
+
+resource "azurerm_servicebus_namespace_authorization_rule" "wisp_converter_servicebus_ns_auth" {
+  count = var.enable_wisp_converter ? 1 : 0
+
+  name         = "${local.project}-servicebus-ns-manager"
+  namespace_id = azurerm_servicebus_namespace.wisp_converter_servicebus[0].id
+
+  listen = true
```

Comment:
perhaps, like event-hub, it would be better to decouple reading and writing

Reply:
It was added an advice as rule to follow